spec/readme.md

```diff
@@ -162,6 +171,15 @@ Unlike a gear's config values, contexts are not guaranteed to exist _or_ have a
 
 Because context values are not namespaced, it is suggested that you use a specific and descriptive name. The `matlab_license_code` example is a good, self-explanatory key that many gears could likely reuse.
 
+
+### API keys
+
+It is possible to interact with the Flywheel data hierarchy using our [python](https://flywheel-io.github.io/core/branches/master/python/getting_started.html), [matlab](https://flywheel-io.github.io/core/branches/master/matlab/getting_started.html), and (slated for an overhaul) [golang](https://github.com/flywheel-io/sdk) SDKs.
+
+Generally, you will want to figure out a script that you like, using your normal user API key, before turning it into a gear. To do this, specify an `api-key` type input as show in the example above, and use that value in your gear script.
+
```

Comment:
Maybe one-sentence call-out that by default the api-key will have write permissions and can be made read-only.

Reply:
It's in the example blob, but I'll add it here to. Good to have in both places